irishlang.py

```diff
@@ -0,0 +1,63 @@
+import math
+import ast
+class irishspell:
+    def __init__(self):
+        self.file1 = open("irish.txt", "r", encoding = "UTF-16")
+        self.corpus1 = self.file1.read()
+        self.frequency_dictionary = ast.literal_eval(self.corpus1)
+        self.file1.close()
+        self.file3 = open("frequency.txt", "r", encoding = "UTF-16")

```

Comment:
Will you upload this file too please?

Reply:
[frequency.txt](https://github.com/maddiekoehler/spellchecker-team5/files/7556849/frequency.txt)